config/cesm/machines/config_machines.xml

```diff
@@ -635,10 +635,10 @@ This allows using a different mpirun command to launch unit tests
         <command name="load">pnetcdf/1.12.1</command>
       </modules>
       <modules mpilib="mpt" compiler="intel">
-        <command name="load">mpt/2.21</command>
+        <command name="load">mpt/2.23</command>
         <!-- known failure in parallel netcdf with mpt, use serial -->
         <command name="load">netcdf/4.7.3</command>
```

Comment:
The known failure message is old, I believe that you can also change this to netcdf-mpi/4.7.4

Reply:
I've removed the old failure message, and changed to netcdf-mpi/4.7.4 and ran a quick B1850 test.